### PR TITLE
flatten input_data parameters

### PR DESCRIPTION
**Pull request by rubenchazarra**
This PR is simply to flatten the input data parameters in the config file, so that they can be accessed by an upstream workflow. 
If config parameters are organized in scopes (as params{input_data{data_file = "abc.tsv"}}) then they cannot be overwritten. So I took input_data parameters out of their scope.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -109,11 +109,11 @@ if(params.data_download.run == "True"){
     }
     
 }else{
-   	REF_10X_DIR = Channel.fromPath(params.input_data.ref_10x_dir).first()
-   	QUERY_10X_DIR = Channel.fromPath(params.input_data.query_10x_dir).first()
-   	UNMELT_SDRF_REF = Channel.fromPath(params.input_data.unmelt_sdrf_ref).first()
-   	UNMELTED_SDRF_QUERY = Channel.fromPath(params.input_data.unmelt_sdrf_query).first()
-    	REF_MARKERS = Channel.fromPath(params.input_data.ref_markers).first()
+   	REF_10X_DIR = Channel.fromPath(params.ref_10x_dir).first()
```
Comment (pinin4fjords) [inline main.nf]: You should use the 'checkIfExists' functionality to make sure you get an error when input dirs are missing (see https://www.nextflow.io/docs/latest/channel.html#frompath)

Comment (rubenchazarra) [reply main.nf]: Perfect I'll add this!

File: nextflow.config [hunk 2]
```diff
@@ -156,7 +154,7 @@ params {
 
 env { 
     CONTROL_WORKFLOW_ROOT = "${baseDir}"
-    CONTROL_WORKFLOW_BRANCH = 'origin/develop'
+    CONTROL_WORKFLOW_BRANCH = 'origin/feature/cross_validation'
```
Comment (pinin4fjords) [inline nextflow.config]: If this is production(ish) code you shouldn't be using a feature branch of the child workflow (it should be parameterised to deal with your use case). Remember that feature branches are ephemeral- we shouldn't expect them to be around for long.

Comment (rubenchazarra) [reply nextflow.config]: Forgot to change this after merging the feature branch, thanks for pointing out.

**Review by pinin4fjords — APPROVED** [review]
Thanks for addressing the comments- good to go @rubenchazarra !

**Review by a-solovyev12 — APPROVED** [review]

**Pull request merged**